NetKAN/SigmaBinary-core.netkan

```diff
@@ -1,18 +1,18 @@
 {
-    "spec_version" : "v1.4",
+    "spec_version" : "v1.5.1",
```

Comment:
This should still be "v1.4". It's the version of the [CKAN spec](https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md) required to accurately read metadata.